build.rs

```diff
@@ -0,0 +1,14 @@
+use cfgrammar::yacc::YaccKind;
+use lrlex::LexerBuilder;
+use lrpar::CTParserBuilder;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let lex_rule_ids_map = CTParserBuilder::<u8>::new_with_storaget()
```

Comment:
I know this is upstream code, but the term `storaget` is unfortunate. I guess it means "storage T", but it's hard not to read it as one word.

Reply:
I thought the same about `reposdir` fwiw. Took a second glance to understand the meaning. An underscore would help, but I'm not going to insist ;)

Reply:
`StorageT` is what it is. `reposdir` is more a matter of taste. We disagree on that ;)

Reply:
`StorageT` is fine (and not at question), but `storaget` is hard to read.

All I'm saying is, sometimes an underscore would help to delimit words.

E.g. `storage_t`.

(I'm not going to insist)

Reply:
You're asking for a change in another repo, so the answer for this PR is definitely "out of scope" :p

Reply:
Yes, I wasn't asking for a change, only commenting.

> I know this is upstream code, but ...